dask_grblas/expr.py

```diff
@@ -15,6 +15,66 @@ def __init__(self, parent, method_name, *args, meta, **kwargs):
         self.kwargs = kwargs
         self._meta = meta
 
+    def _matmult(self, meta, updating=None, mask=None, accum=None):
```

Comment:
It might be worth considering reimplementing `_matmult()` in a way similar to that of the more efficient `dask.array.matmul()` (See [this link](https://github.com/dask/dask/issues/6874#issue-748092230)).

I didn't do it now because it seems to involve adding new dimensions to matrices, and I'm not quite sure how `grblas` would handle that.